app/models/orders_package.rb

```diff
@@ -162,4 +164,14 @@ def update_designation_of_package
   def destroy_stockit_record(operation)
     StockitSyncOrdersPackageJob.perform_now(package.id, self.id, operation) unless package.is_singleton_package?
   end
+
+  def valid_package_state?
+    package.received? && package.inventory_number? && package.locations.present?
```

Comment:
could we split these up into separate validations?

    validates :inventory_number, :presense
    validates :locations, :presense